lib/stripe/api.ex

```diff
@@ -135,6 +135,20 @@ defmodule Stripe.API do
   """
   @spec request(body, method, String.t(), headers, list) ::
           {:ok, map} | {:error, Stripe.Error.t()}
+  def request(body, :get, endpoint, headers, opts) do
```

Comment:
I do.  On its own, the `:get` request change fixed the upcoming invoice test (you can verify the failure by testing the current `master` against a `v0.30.x` version of stripe-mock), and I've tested this branch against the live (well, test) Stripe API as well.

Our Stripe logs verify parameters coming in now as `Request query parameters` instead of `Request POST body` for GET requests.